pkg/controllers/user/logging/utils/embedded.go

```diff
@@ -26,7 +26,7 @@ const (
 	running = "Running"
 )
 
-func CreateEmbeddedTarget(dep rv1beta2.DeploymentInterface, sa rv1.ServiceAccountInterface, se rv1.ServiceInterface, ro rrbacv1.RoleInterface, rb rrbacv1.RoleBindingInterface, namespace string) error {
+func CreateOrUpdateEmbeddedTarget(dep rv1beta2.DeploymentInterface, sa rv1.ServiceAccountInterface, se rv1.ServiceInterface, ro rrbacv1.RoleInterface, rb rrbacv1.RoleBindingInterface, namespace string, obj *v3.ClusterLogging) error {
 	// create es deployment
 	_, err := dep.Controller().Lister().Get(loggingconfig.EmbeddedESName, loggingconfig.EmbeddedESName)
```

Comment:
Looks like we are using wrong namespace here so we'll always go into the create procedure.
Should it be `loggingconfig.EmbeddedESName`->`loggingconfig.LoggingNamespace` for the namespace?